### DISCUSSION TITLE
One click boost option

### DISCUSSION THREAD

**Discussion by dougfinnie** [tuskyapp/Tusky#3057] [Ideas]
Something I really like about the web interface is the single click boost. No confirmation, it just happens.

I totally understand why some might prefer to confirm, but can we jave an option that makes it just one click?

**Comment by charlag** [comment]
There is an option

**Reply by dougfinnie** [reply]
Well this is embarassing 😳 I even installed the nightly and still didnt see it.

**Reply by AkazaRenn** [reply]
> Well this is embarassing 😳 I even installed the nightly and still didnt see it.

Preference - Show confirmation dialogue before boosting
